spoj_practice/spoj2/spoj2.py

```diff
@@ -51,12 +38,14 @@ def find_primes_between (n, m):
     # m = numbers[0]
     # n = numbers[1]
     # instead, do:
-    m, n = stdin.readline().split(' ')
-    m = int(m)
-    n = int(n)
+    m, n = map(int, stdin.readline().split(' '))
     prime = []
```

Comment:
because i want to make sure it's an empty list. 
if i just had primes = find_primes_between(n, m) 
it would rewrite certain values at the indicies between n and m per iteration of the loop, but it would be possible that not all the values are overwritten, thus I would have values that overflowed from the previous loop iteration for a different set of (n, m) which I don't want. thus I re-define it as empty list every time to "clear" it first.